examples/views/button_roles.py

```diff
@@ -83,15 +83,16 @@ async def post(self, ctx: commands.Context):
 
     @commands.Cog.listener()
     async def on_ready(self):
-        """This function is called every time the bot restarts.
+        """
+        This function is called every time the bot restarts.
```

Comment:
```suggestion
        """This function is called every time the bot restarts.
```
Consistency with project code style.